scripts/install-helpers/baremetal-coco/install.sh

```diff
@@ -690,6 +831,7 @@ if [ "$SKIP_NFD" = false ]; then
     tdx)
         create_intel_node_feature_rules || exit 1
         deploy_intel_device_plugins || exit 1
+        deploy_intel_dcap || exit 1
```

Comment:
IMHO it would be clearer to use the _install/uninstall_ wording instead of _deploy/uninstall_ but this can be fixed later.

Reply:
I won't change this for now, as the `deploy` pattern was there before my patches, and I'm just following that.

Later on we can decide to have a better naming, and change it all over the place, at once.